src/platforms/common/configuration/options.mdx

```diff
@@ -379,6 +396,12 @@ The number of characters after which the values containing text in the event pay
 
 <ConfigKey name="with-locals" supported={["python"]}>
```

Comment:
I noticed this option was also deprecated: https://github.com/getsentry/sentry-python/blob/master/sentry_sdk/client.py#L82C20-L82C20. So I added the alert here too!